alerts.tf

```diff
@@ -0,0 +1,19 @@
+module "wa-exception-alert" {
+  source = "git@github.com:hmcts/cnp-module-metric-alert"
+  location = "${var.location}"
+
+  app_insights_name = "wa-${var.env}"
+
+  alert_name = "wa-dlq-alert"
+  alert_desc = "Triggers when a message falls into the Dead Letter Queue, works with 5 minute poll in wa-${var.env}."
+  app_insights_query = "union traces, exceptions | where customDimensions[\"LoggingLevel\"] == \"WARN\" and customDimensions[\"Logger Message\"] contains \"dead lettered\" | sort by timestamp desc"
```

Comment:
looks ok to me assuming the messages we've put specifically from the DLQ have "dead lettered" etc.